client/src/components/menuBar/menuBar.js

```diff
@@ -34,6 +35,11 @@ export default function MenuBar() {
     setAnchorEl(null);
   };
 
+  const _openModal = () => {
+    handleClose();
+    openModal();
+  }
+
```

Comment:
This implementation was pretty wonky, since we wanted modal to be on the same level as the `menubar`, i.e. `MenuBar` and `ProfileModal` are siblings, so we had to do this weird callback prop drilling thing since the boolean/state/variable controlling if the modal was open is contained in their parent